pack/tic/def_encounter.json

```diff
@@ -0,0 +1,598 @@
+		"back_illustrator": "Jorge Ramos Márquez",
+		"back_name": "Devil Reef",
+		"clues": 0,
+		"code": "07169",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 7,
+		"faction_code": "mythos",
+		"illustrator": "Ryan Barger",
+		"name": "Lonely Isle",
+		"pack_code": "def",
+		"position": 169,
+		"quantity": 1,
+		"shroud": 5,
+		"text": "<b>Forced</b> - After Lonely Isle is revealed: Put 2 random set-aside Tidal Tunnels into play, each adjacent to Lonely Isle (but not to one another).",
+		"traits": "Ocean. Island.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"...as I glimpsed it I could not help thinking of all the hideous legends I had heard in the last twenty-four hours - legends which portrayed this ragged rock as a veritable gateway to realms of unfathomed horror and inconceivable abnormality.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
+		"back_illustrator": "Jorge Ramos Márquez",
+		"back_name": "Devil Reef",
+		"clues": 0,
+		"code": "07170",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 8,
+		"faction_code": "mythos",
+		"illustrator": "Carlos Palma Cruchaga",
+		"name": "Hidden Cove",
+		"pack_code": "def",
+		"position": 170,
+		"quantity": 1,
+		"shroud": 3,
+		"text": "<b>Forced</b> - After you reveal Hidden Cove: Put 1 set-aside Tidal Tunnel into play adjacent to Hidden Cove, then put 1 set-aside Unfathomable Depths into play adjacent to that Tidal Tunnel.",
+		"traits": "Ocean. Island.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"...as I glimpsed it I could not help thinking of all the hideous legends I had heard in the last twenty-four hours - legends which portrayed this ragged rock as a veritable gateway to realms of unfathomed horror and inconceivable abnormality.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
+		"back_illustrator": "Jorge Ramos Márquez",
+		"back_name": "Devil Reef",
+		"clues": 0,
+		"code": "07171",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 9,
+		"faction_code": "mythos",
+		"illustrator": "Dual Brush Studios",
+		"name": "Waveworn Island",
+		"pack_code": "def",
+		"position": 171,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After you reveal Waveworn Island: Increase its flood level. Put 1 random set-aside Unfathomable Depths and 2 random set-aside Tidal Tunnels into play, each adjacent to Waveworn Island (but not to one another).",
+		"traits": "Ocean. Island.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"...as I glimpsed it I could not help thinking of all the hideous legends I had heard in the last twenty-four hours - legends which portrayed this ragged rock as a veritable gateway to realms of unfathomed horror and inconceivable abnormality.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```

Comment:
Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).